config/routes.rb

```diff
@@ -6,14 +6,24 @@
   # Can be used by load balancers and uptime monitors to verify that the app is live.
   get "up" => "rails/health#show", as: :rails_health_check
 
+  #Define Api routes
+  namespace :api do
+    namespace :v1 do
+      resources :posts, only: [:index]
+      resources :comments, only: [:index, :create]
+    end
+  end
```

Comment:
**OPTIONAL** - You did well so far, but to get the expected endpoints according to the project requirements, I suggest you configure your project like I did below: Please note that it is important you get your routes correctly configured to ensure your app works as expected

```ruby 
    namespace :api do
        namespace :v1 do
           resources :users, only[:show] do
               resources :posts, only: [:index] do
                    resources :comments, only: [:index, :create]
               end
           end
         end
     end